src/dstack/_internal/core/backends/digitalocean_base/models.py

```diff
@@ -0,0 +1,47 @@
+from typing import Annotated, List, Literal, Optional, Union
+
+from pydantic import Field
+
+from dstack._internal.core.models.common import CoreModel
+
+
+class BaseDigitalOceanAPIKeyCreds(CoreModel):
+    type: Annotated[Literal["api_key"], Field(description="The type of credentials")] = "api_key"
+    api_key: Annotated[str, Field(description="The API key")]
+
+
+AnyBaseDigitalOceanCreds = BaseDigitalOceanAPIKeyCreds
+BaseDigitalOceanCreds = AnyBaseDigitalOceanCreds
+
+
+class BaseDigitalOceanBackendConfig(CoreModel):
+    type: Annotated[
+        Literal["amddevcloud", "digitalocean"],
+        Field(description="The type of backend"),
+    ] = "digitalocean"
```

Comment:
The reference is generated incorrectly - it says that `type` must be set to `digitalocean` for the `amddevcloud` backend.

<img width="624" height="290" alt="reference" src="https://github.com/user-attachments/assets/fc248113-34a8-4eaf-befe-0b2456b34bb6" />


We may need to look into updating the reference generating script, `gen_schema_reference.py`, but the easiest way to fix this now is to remove the default.

```suggestion
    type: Annotated[
        Literal["amddevcloud", "digitalocean"],
        Field(description="The type of backend"),
    ]
```